Sources/Parsing/Parsers/Many.swift

```diff
@@ -29,49 +29,49 @@ import Foundation
 /// precondition(input == "")
 /// precondition(output == 6)
 /// ```
-public struct Many<Upstream, Result, Separator>: Parser
+public struct Many<Element, Result, Separator>: Parser
```

Comment:
Other ideas: `Each` or `EachParser`?

Also...this change _is_ breaking.

Introducing both a deprecated type alias _and_ a deprecated property seemed a bit heavy, but let me know and I'll add them!